### PR TITLE
sonic show interface alias

### PR DESCRIPTION
**Pull request by rishabh5j**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT
sonic_linux_show_interfaces_alias.template
Sonic
"show interfaces alias"

##### SUMMARY

Addition of new template

```

```

### CODE REVIEW

**Comment by sidtyagi** [comment]
bump the version number please

File: templates/sonic_linux_show_interfaces_alias.template [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Value NAME (\S+)
```
Comment (sidtyagi) [inline templates/sonic_linux_show_interfaces_alias.template]: hopefully this been tested against the python script has well ?

Comment (rishabh5j) [reply templates/sonic_linux_show_interfaces_alias.template]: @sidtyagi  yup. As a standard, the test parsed file should always be collected from the script produced output, that's what is done here.

**Comment by rishabh5j** [comment]
> bump the version number please

done.

**Pull request merged**